### PR TITLE
Develop v0.9.28

### PR DESCRIPTION
**Pull request by devdanco**

### CODE REVIEW

File: runtime/mangata-rococo/src/lib.rs [hunk 1]
```diff
@@ -881,6 +881,8 @@ impl pallet_elections_phragmen::Config for Runtime {
 	type DesiredMembers = DesiredMembers;
 	type DesiredRunnersUp = DesiredRunnersUp;
 	type TermDuration = TermDuration;
+	type MaxVoters = MaxVoters;
+	type MaxCandidates = MaxCandidates;
```
Comment (iStrike7) [inline runtime/mangata-rococo/src/lib.rs:885]: Where is this coming from?

Comment (iStrike7) [reply runtime/mangata-rococo/src/lib.rs:885]: The parameter_types block on line 778 is defunct. Please remove it and define MaxCandidates where you have defined max voters.

File: runtime/mangata-rococo/src/lib.rs [hunk 2]
```diff
@@ -858,6 +857,7 @@ parameter_types! {
 	pub const TermDuration: BlockNumber = 120 * DAYS;
 	pub const DesiredMembers: u32 = 9;
 	pub const DesiredRunnersUp: u32 = 7;
+	pub const MaxVoters: u32 = 10 * 1000;
```
Comment (iStrike7) [inline runtime/mangata-rococo/src/lib.rs:860]: How was this value decided?

**Review by iStrike7 — CHANGES_REQUESTED** [review]

**Review by iStrike7 — APPROVED** [review]

File: node/src/service.rs [hunk 3]
```diff
@@ -353,10 +356,6 @@ where
 		bool,
 	) -> Result<Box<dyn ParachainConsensus<Block>>, sc_service::Error>,
 {
-	if matches!(parachain_config.role, Role::Light) {
```
Comment (mateuszaaa) [inline node/src/service.rs:356]: dont we need that anymore?

File: runtime/mangata-kusama/src/lib.rs [hunk 4]
```diff
@@ -867,6 +866,7 @@ parameter_types! {
 	pub const TermDuration: BlockNumber = 120 * DAYS;
 	pub const DesiredMembers: u32 = 9;
 	pub const DesiredRunnersUp: u32 = 7;
+	pub const MaxVoters: u32 = 10 * 1000;
```
Comment (mateuszaaa) [inline runtime/mangata-kusama/src/lib.rs:869]:
```suggestion
	pub const MaxVoters: u32 = 10_000_u32;;
```

File: runtime/mangata-rococo/src/lib.rs [hunk 5]
```diff
@@ -858,6 +848,8 @@ parameter_types! {
...
```
Comment (mateuszaaa) [inline runtime/mangata-rococo/src/lib.rs:851]:
```suggestion
	pub const MaxVoters: u32 = 10_000_u32;
```

File: runtime/mangata-kusama/src/lib.rs (hunk 4 above)
Comment (mateuszaaa) [reply runtime/mangata-kusama/src/lib.rs:869]: @iStrike7 can you confirm that 10000 is proper value ?

**Review by mateuszaaa — CHANGES_REQUESTED** [review]
few minors comments

**Review by mateuszaaa — CHANGES_REQUESTED** [review]
please squash your changes to single commit or fix the history to be more readable ;)

**Comment by devdanco** [comment]
@mateuszaaa when we merge we can squash those commit :)

File: runtime/mangata-kusama/src/lib.rs (hunk 4 above)
Comment (devdanco) [reply runtime/mangata-kusama/src/lib.rs:869]: @mateuszaaa here https://github.com/paritytech/substrate/blob/polkadot-v0.9.28/bin/node/runtime/src/lib.rs#L969

File: node/src/service.rs (hunk 3 above)
Comment (devdanco) [reply node/src/service.rs:356]: Role::Light was removed from 28 .. therefore we do not need this anymore .. @mateuszaaa

**Pull request merged**